### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jeniferh**
Add changes since v0.1 to update the operator image

### CODE REVIEW

**Review by jtaleric — APPROVED** [review]
LGTM, not sure of a more "proper way" either.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -124,6 +124,7 @@ Any changes to the [roles](roles/) tree or to the [playbook](playbook.yml) file
 The benchmark-operator container can be built using [podman](https://podman.io/) and pushed to a public repository.
 The public repository could be [quay](https://quay.io) in which case you'll need to:
 
+#### v0.1
```
Comment (rsevilla87) [inline CONTRIBUTING.md:127]: Hey @jeniferh, the docs for the v0.1 tag are correct, I think we shouldn't include the v0.1 section in the master branch,

I'd just move the comments from the "v0.1" you've created here to the "The operator container image" section

**Review by rsevilla87 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (jeniferh) [reply CONTRIBUTING.md:127]: Good point, I will adjust so the master version only documents the current method since the v0.1 tag covers the old method.

**Review by rsevilla87 — APPROVED** [review]
LGTM!

**Review by sjug — APPROVED** [review]
/lgtm

**Pull request merged**